MediaGalleryUi/etc/db_schema.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<schema xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Setup/Declaration/Schema/etc/schema.xsd">
+    <table name="media_gallery_asset">
+        <index referenceId="MEDIA_GALLERY_ASSET_TITLE" indexType="btree">
+            <column name="title"/>
```

Comment:
This change is done in https://github.com/magento/magento2/pull/27499 Please revert the db changes